src/jit/hwintrinsicxarch.cpp

```diff
@@ -905,6 +905,16 @@ GenTree* Compiler::impSSEIntrinsic(NamedIntrinsic        intrinsic,
             break;
         }
 
+        case NI_SSE_StaticCast:
+        {
+            // We fold away the static cast here, as it only exists to satisfy
+            // the type system. It is safe to do this here since the retNode type
+            // and the signature return type are both TYP_SIMD16.
```

Comment:
Can an assert be added for this comment as well?  (assert the return type)

Reply:
Probably. Will update.

Reply:
Fixed.